common/logging.edl

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+/*
+**==============================================================================
+**
+** logging.edl:
+**
+**     This file declares internal ECALLs/OCALLs used by liboehost/liboecore
+**     for logging to the host.
+**
+**==============================================================================
+*/
+
+enclave
+{
+
+    trusted
+    {
+        public void oe_log_init_ecall(
+            [in, string] const char* enclave_path,
+            uint32_t log_level);
+    };
+
+    untrusted
+    {
+        void oe_log_ocall(
+            uint32_t log_level,
+            [in, string] const char* message);
+
+        // Write a string to the console. Write to STDOUT if device=0. Write
+        // to STDERR if device=1. Write strnlen(str, maxlen) bytes.
+        void oe_write_ocall(
```

Comment:
Not for this PR, but we should figure out if the log APIs now have dependencies on syscall APIs and just use those directly perhaps.

Reply:
I think the reason oe_write_ocall exists is so that we can do logging without depending on the syscall APIs.